libfwupdplugin/fu-security-attrs.h

```diff
@@ -14,6 +14,37 @@
 
+#define FU_SECURITY_ATTR_ID_ACPI_DMAR			"org.fwupd.hsi.AcpiDmar"		/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_ENCRYPTED_RAM		"org.fwupd.hsi.EncryptedRam"		/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_FWUPD_ATTESTATION		"org.fwupd.hsi.Fwupd.Attestation"	/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_FWUPD_PLUGINS		"org.fwupd.hsi.Fwupd.Plugins"		/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_FWUPD_UPDATES		"org.fwupd.hsi.Fwupd.Updates"		/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_INTEL_BOOTGUARD_ENABLED	"org.fwupd.hsi.IntelBootguard.Enabled"	/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_INTEL_BOOTGUARD_VERIFIED	"org.fwupd.hsi.IntelBootguard.Verified"	/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_INTEL_BOOTGUARD_ACM		"org.fwupd.hsi.IntelBootguard.Acm"	/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_INTEL_BOOTGUARD_POLICY	"org.fwupd.hsi.IntelBootguard.Policy"	/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_INTEL_BOOTGUARD_OTP		"org.fwupd.hsi.IntelBootguard.Otp"	/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_INTEL_CET_ENABLED		"org.fwupd.hsi.IntelCet.Enabled"	/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_INTEL_CET_ACTIVE		"org.fwupd.hsi.IntelCet.Active"		/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_INTEL_SMAP			"org.fwupd.hsi.IntelSmap"		/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_IOMMU			"org.fwupd.hsi.Iommu"			/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_KERNEL_LOCKDOWN		"org.fwupd.hsi.Kernel.Lockdown"		/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_KERNEL_SWAP			"org.fwupd.hsi.Kernel.Swap"		/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_KERNEL_TAINTED		"org.fwupd.hsi.Kernel.Tainted"		/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_MEI_MANUFACTURING_MODE	"org.fwupd.hsi.Mei.ManufacturingMode"	/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_MEI_OVERRIDE_STRAP		"org.fwupd.hsi.Mei.OverrideStrap"	/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_MEI_VERSION			"org.fwupd.hsi.Mei.Version"		/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_SPI_BIOSWE			"org.fwupd.hsi.Spi.Bioswe"		/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_SPI_BLE			"org.fwupd.hsi.Spi.Ble"			/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_SPI_SMM_BWP			"org.fwupd.hsi.Spi.SmmBwp"		/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_SUSPEND_TO_IDLE		"org.fwupd.hsi.SuspendToIdle"		/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_SUSPEND_TO_RAM		"org.fwupd.hsi.SuspendToRam"		/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_TPM_RECONSTRUCTION_PCR0	"org.fwupd.hsi.Tpm.ReconstructionPcr0"	/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_TPM_VERSION_20		"org.fwupd.hsi.Tpm.Version20"		/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_UEFI_SECUREBOOT		"org.fwupd.hsi.Uefi.SecureBoot"		/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_INTEL_DCI_ENABLED		"org.fwupd.hsi.IntelDci.Enabled"	/* Since: 1.5.0 */
+#define FU_SECURITY_ATTR_ID_INTEL_DCI_LOCKED		"org.fwupd.hsi.IntelDci.Locked"		/* Since: 1.5.0 */
+
```

Comment:
How about if this whole section is moved into `libfwupdplugin/fu-security-attrs-private.h` and that's not installed into the filesystem?  It's then only usable by plugins in the tree, and we don't need to worry about it being supported forever.  Once my other concerns are assayed and more than Intel are on board we can make it installed in tree again?

Reply:
> moved into libfwupdplugin/fu-security-attrs-private.h

That's an even better idea. Will fix, but have a few hours of meetings first.

Reply:
> How about if this whole section is moved into libfwupdplugin/fu-security-attrs-private.h

That worked wonderfully, fixed.